addons/hr_recruitment_event/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_applicant_event_report,access_hr_applicant_event_report,model_hr_applicant_event_report,base.group_user,1,0,0,0
```

Comment:
Why exposing applicant/event data to all employees (even those without access to both models) ?